spec/files/auth/catalogue_shared_auth_hash_no_email.json

```diff
@@ -54,7 +54,8 @@
       "name": "Yetrina Shared",
       "preferred_username": "ybattad_shared",
       "given_name": "Yetrina",
-      "family_name": "Shared"
+      "family_name": "Shared",
+      "folio_id": "folio_id"
```

Comment:
Is "folio_active" property only present if true, or just omitted from some of the spec files cos it's not needed for tests

Reply:
I just omitted it for the tests.